src/main/java/com/spotinst/sdkjava/model/bl/aws/managed/instance/Tasks.java

```diff
@@ -0,0 +1,167 @@
+package com.spotinst.sdkjava.model.bl.aws.managed.instance;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.enums.SchedulingTaskTypeEnum;
+import com.spotinst.sdkjava.model.RecurrenceFrequencyEnum;
```

Comment:
Define this class in enums and use from there, so that code is correct as per latest folder structure.

Reply:
created RecurrenceFrequencyEnum in Enums folder